crates/biome_js_analyze/tests/specs/nursery/noReactForwardRef/react19-catalog/catalog.js

```diff
@@ -0,0 +1,7 @@
+/* should generate diagnostics with pnpm catalog */
```

Comment:
These comments are magic comments that tell our infra to make sure these do or don't emit diagnostics, but they need to be exact matches in order for it to work.

```suggestion
/* should generate diagnostics */
```